sale_delivery_recreate/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 Sergio Corato <https://github.com/sergiocorato>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Sale Recreate Delivery",
+    "summary": "Add ability to recreate delivery in sale order",
+    "version": "12.0.1.0.0",
+    "development_status": "Beta",
+    "category": "Sale",
+    "website": "https://github.com/OCA/sale-workflow",
+    "author": "Sergio Corato, "
+              "Odoo Community Association (OCA)",
+    "maintainers": ["sergiocorato"],
+    "license": "AGPL-3",
+    "installable": True,
+    "depends": [
+        "purchase_stock",
```

Comment:
@sergiocorato Is this really needed ?

Reply:
@rousseldenis without `purchase_stock` no buy route, so no procurement is possible and `_get_rule` gives:
`odoo.exceptions.UserError: ('No procurement rule found in location "WH/Stock" for product "[E-COM11] Cabinet with Doors".\n Check routes configuration.', '') `

Reply:
> @rousseldenis without `purchase_stock` no buy route, so no procurement is possible and `_get_rule` gives:
> `odoo.exceptions.UserError: ('No procurement rule found in location "WH/Stock" for product "[E-COM11] Cabinet with Doors".\n Check routes configuration.', '') `

In tests or in real ?

Reply:
@rousseldenis Ok, this is a good point. I removed the exception to include case of purchase_stock not installed, after testing with purchase_stock installed I'll launch removing it. Thanks.

Reply:
I fixed and extended tests to cover internal pull of stock too.

Reply:
@rousseldenis done, thanks!